CHANGELOG.md

```diff
@@ -23,6 +23,22 @@ CHANGELOG
 
 Swift 5.0
 ---------
+* [SR-9494][]:
+
+  Starting from `-swift-version 5` implicit `@autoclosure` function type
+  forwarding has been disabled and new diagnostic has been added suggesting
```

Comment:
comma: "Starting from -swift-version
comma: "has been disabled,"
missing 'a': "and a new diagnostic"